ReactWindows/ReactNative/Views/View/ReactViewManager.cs

```diff
@@ -0,0 +1,270 @@
+        /// The exported custom bubbling event types.
+        /// </summary>
+        public override IReadOnlyDictionary<string, object> ExportedCustomBubblingEventTypeConstants
+        {
+            get
+            {
+                return new Dictionary<string, object>()
+                {
+                    {
+                        "topDragEnter",
+                        new Dictionary<string, object>()
+                        {
+                            {
+                                "phasedRegistrationNames",
+                                new Dictionary<string, string>()
+                                {
+                                    { "bubbled" , "onDragEnter" },
+                                    { "captured" , "onDragEnterCapture" }
+                                }
+                            }
+                        }
+                    },
+                    {
+                        "topDragOver",
+                        new Dictionary<string, object>()
+                        {
+                            {
+                                "phasedRegistrationNames",
+                                new Dictionary<string, string>()
+                                {
+                                    { "bubbled" , "onDragOver" },
+                                    { "captured" , "onDragOverCapture" }
+                                }
+                            }
+                        }
+                    },
+                    {
+                        "topDrop",
+                        new Dictionary<string, object>()
+                        {
+                            {
+                                "phasedRegistrationNames",
+                                new Dictionary<string, string>()
+                                {
+                                    { "bubbled" , "onDrop" },
+                                    { "captured" , "onDropCapture" }
+                                }
+                            }
+                        }
+                    },
+                    {
+                        "topDragLeave",
+                        new Dictionary<string, object>()
+                        {
+                            {
+                                "phasedRegistrationNames",
+                                new Dictionary<string, string>()
+                                {
+                                    { "bubbled" , "onDragLeave" },
+                                    { "captured" , "onDragLeaveCapture" }
+                                }
+                            }
+                        }
+                    },
+                };
+            }
+        }
+
+        /// <summary>
+        /// Enables the Canvas as a drop target.
+        /// </summary>
+        [ReactProp("allowDrop")]
+        public void SetAllowDrop(BorderedCanvas view, bool allowDrop)
+        {
+            view.AllowDrop = allowDrop;
+
```

Comment:
If we're going to expose `AllowDrop`, should we also expose `CanDrag`?

Reply:
This sounds like a different feature: dragging the view vs dropping files onto the view.

Reply:
Fair point. Dragging views is easily implemented with React.js anyway.

---
In reply to: [159530752](https://github.com/Microsoft/react-native-windows/pull/1549#discussion_r159530752) [](ancestors = 159530752)